suite-common/wallet-config/src/types.ts

```diff
@@ -97,3 +99,5 @@ export type Network = {
 export type Networks = {
     [key in NetworkSymbol]: Network;
 };
+
+export type NetworkAccountCollection = Array<Account & { accountType: AccountType }>;
```

Comment:
:magic_wand: :sparkles:  https://github.com/trezor/trezor-suite/pull/14109/commits/1970728c7c7b914d91c9bb53077fd7c67acec696

I've done the same thing for network symbol. Previously, this would be allowed for Networks, but now it gives nice error:
![ts magic Networks](https://github.com/user-attachments/assets/d4e0715e-596f-4126-9bde-5bfe455bd59e)

And with this change, also for Accounts:
![ts magic Accounts](https://github.com/user-attachments/assets/a506495e-619d-4c5c-bcce-82dafb6bcf1b)